### ISSUE TITLE
[BUG] Shutdown doesn't finish properly

### ISSUE THREAD

**Issue by daywalker90** [lnbits/lnbits#1485]
**Describe the bug**
Lnbits seems to stop properly but then finally does not "return".

**To Reproduce**
1. start lnbits with poetry run lnbits --host 0.0.0.0 --port 5000
2. Wait for full startup
3. Ctrl+C

**Expected behavior**
After logs say shutdown complete it should return back to cli.

**Screenshots**

**Desktop (please complete the following information):**
 - LNbits version: 9840a8b15417df74fe926de85a82efd9e8664a31 but this happend as long as i'm using lnbits (prob a year+)
 - Database postgres 14.5
 - Debian 11 up-to-date

**Additional context**
```
2023-02-13 18:09:16.81 | INFO | Started server process [793121]
2023-02-13 18:09:16.81 | INFO | Waiting for application startup.
2023-02-13 18:09:18.24 | INFO | ✔️ All migrations done.
2023-02-13 18:09:18.24 | INFO | Starting LNbits
2023-02-13 18:09:18.24 | INFO | Host: 0.0.0.0
2023-02-13 18:09:18.24 | INFO | Port: 5000
2023-02-13 18:09:18.24 | INFO | Debug: False
2023-02-13 18:09:18.24 | INFO | Site title: LNbits
2023-02-13 18:09:18.24 | INFO | Funding source: CoreLightningWallet
2023-02-13 18:09:18.24 | INFO | Data folder: ./data
2023-02-13 18:09:18.24 | INFO | Git version: 9840a8b15417df74fe926de85a82efd9e8664a31
2023-02-13 18:09:18.24 | INFO | Database: PostgreSQL
2023-02-13 18:09:18.24 | INFO | Service fee: 0.0
2023-02-13 18:09:18.28 | INFO | ✔️ Backend CoreLightningWallet connected and with a balance of XXXX msat.
2023-02-13 18:09:18.29 | INFO | Application startup complete.
INFO:uvicorn.error:Application startup complete.
2023-02-13 18:09:18.29 | INFO | Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
INFO:uvicorn.error:Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
2023-02-13 18:09:18.29 | INFO | Task: checking all pending payments (incoming=True, outgoing=True) of last 15 days
2023-02-13 18:09:18.30 | WARNING | supplied an invalid checking_id: af15c03d52223e45e395e5faf92eb75006c066c61244e795373156bdab12a05d
2023-02-13 18:09:18.30 | INFO | Task: pending check finished for 1 payments (took 0.010 s)
2023-02-13 18:09:18.30 | INFO | Task: expired invoice deletion finished (took 0.001 s)
^C
Aborted!
2023-02-13 18:09:30.93 | INFO | Shutting down
INFO:uvicorn.error:Shutting down
2023-02-13 18:09:31.03 | INFO | Waiting for application shutdown.
INFO:uvicorn.error:Waiting for application shutdown.
2023-02-13 18:09:31.03 | INFO | Application shutdown complete.
INFO:uvicorn.error:Application shutdown complete.
2023-02-13 18:09:31.03 | INFO | Finished server process [793121]
INFO:uvicorn.error:Finished server process [793121]
^CError in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/usr/lib/python3.9/multiprocessing/popen_fork.py", line 27, in poll
    pid, sts = os.waitpid(self.pid, flag)
KeyboardInterrupt
```

notice the second CTRL+C at that point it would not return and take forever/timeout.

**Comment by talvasconcelos** [comment]
Possible fix in #1858 in next release!

Consider closing the issue if it fixes

**Comment by daywalker90** [comment]
it seems indeed fixed, good job everybody
